pkg/controller/podautoscaler/replica_calculator.go

```diff
@@ -334,7 +334,7 @@ func (c *ReplicaCalculator) GetExternalMetricReplicas(currentReplicas int32, tar
 	if err != nil {
 		return 0, 0, time.Time{}, err
 	}
-	metrics, timestamp, err := c.metricsClient.GetExternalMetric(metricName, namespace, metricLabelSelector)
```

Comment:
SA4006: this value of `timestamp` is never used

Reply:
same reason with https://github.com/kubernetes/kubernetes/pull/110811#discussion_r907938149

https://github.com/kubernetes/kubernetes/blob/223addae4242ef754a5e185c401d8b95dd30cc83/pkg/controller/podautoscaler/replica_calculator.go#L332-L349

Reply:
yep